config/vufind/config.ini

```diff
@@ -799,10 +799,11 @@ authors         = Wikipedia
 ;europeanaAPI = INSERTKEY
 
 ; Geographic Display
-; recordMap can be set to 'openlayers';
-; 'openlayers' will display point and rectangle features. Default setting
-; (see import/marc_local.properties for more information).
+; These configuration settings have been superseded by the geofeatures.ini file.
+; See the [MapTab] section of the geofeatures.ini file for more information.
+;
```

Comment:
If geofeatures.ini is the master configuration now, I'd suggest deleting all extra configuration comments that duplicate information found in that file. It makes the documentation easier to maintain if we only have any given piece of it in one place at a time... The "see for more information" note is helpful, but the rest can probably be deleted.

Reply:
Makes perfect sense to me. I'll delete all the unnecessary configuration comments and leave the "see for more information..."note in place.

Reply:
Should we also remove all of the legacy configuration settings too and only read from the geofeatures.ini file (rather than from config.ini and searches.ini and then geofeatures.ini)? That could streamline the code a lot if we did this.

Reply:
As long as geofeatures.ini offers the same flexibility as the old code did (and I'm pretty sure it does), I don't mind removing the legacy locations... but if we do, we'll need to be sure to document it, and also add support to VuFind\Config\Upgrader to detect the old settings and move them to the new locations. So ultimately it's a question of whether it's more work to write the upgrade code or retain the legacy compatibility.